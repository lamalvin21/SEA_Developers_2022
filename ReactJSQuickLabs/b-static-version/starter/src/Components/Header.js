import React from 'react'
import logoImage from './images/qalogo.svg'

function Header() {
  return (
        <header>
            <nav className="navbar navbar-expand-sm">
                <a href="https://www.qa.com" target="_blank" rel="noopener noreferrer" className="navbar-brand">
                    <img
                        src={logoImage}
                        alt="QA Ltd"
                        width="100"
                        className="d-inline-block align-top"
                    />
                </a>
                <a href="/" className="navbar-brand">
                    <h1>Todo App</h1>
                </a>
            </nav>
        </header>
    )
}

export default Header