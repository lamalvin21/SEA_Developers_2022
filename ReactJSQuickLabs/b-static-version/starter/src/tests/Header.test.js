import React from 'react'
import { create } from 'react-test-renderer'
import Header from '../Components/Header'

test('Matches the snapshot', () => {
    const header = create(<Header />)
    expect(header.toJSON()).toMatchSnapshot()
});
