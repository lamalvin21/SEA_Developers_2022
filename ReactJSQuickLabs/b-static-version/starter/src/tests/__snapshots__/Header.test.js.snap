// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches the snapshot 1`] = `
<header>
  <nav
    className="navbar navbar-expand-sm"
  >
    <a
      className="navbar-brand"
      href="https://www.qa.com"
      rel="noopener noreferrer"
      target="_blank"
    >
      <img
        alt="QA Ltd"
        className="d-inline-block align-top"
        src="qalogo.svg"
        width="100"
      />
    </a>
    <a
      className="navbar-brand"
      href="/"
    >
      <h1>
        Todo App
      </h1>
    </a>
  </nav>
</header>
`;
